cypress/integration/User Management/BM_00_ParentInstructorView.js

```diff
@@ -0,0 +1,60 @@
+describe("Test should load our courses from course.json into course management, check to see if parent's student is enrolled in a course, then view instructor account", () => {
```

Comment:
This should be a high level description of the entire test flow so you shouldn't include "course.json". I'm honestly not sure what you're testing for here. Are you testing your course management flows from the parent's portal? don't combine both parents and instructor portal tests, these should be separate tests.